### DISCUSSION TITLE
Statamic forms

### DISCUSSION THREAD

**Discussion by ottz0** [statamic/cms#7741] [General]
I need to setup some forms in `Statamic` where I can use a `Zapier webhook` to pass onto `Salesforce` the submissions.

I'm also using vue.js so when the form is submitted I want to send GA4 customer tracking conversions. 

Reading the `Statamic` docs, i'm a little confused on how and where to do this. Do I set up a form in the CP>tools>forms and add a new form there?

Or do I use the `Publish forms` and try to handle the submission via `AJAX`?

**Comment by ryanmitchell** [comment]
Publish forms are for creating editing views in the control panel.

You want to set up a form using the CP -> Tools -> Forms area
